cmd/autocomplete/log_drains_remove.go

```diff
@@ -0,0 +1,30 @@
+package autocomplete
+
+import (
+	"fmt"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/urfave/cli"
+	"gopkg.in/errgo.v1"
+)
+
+func LogDrainsRemoveAutoComplete(c *cli.Context) error {
+	appName := CurrentAppCompletion(c)
+	if appName == "" {
+		return nil
+	}
+
+	client, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+	drains, err := client.LogDrainsList(appName)
+	if err == nil {
```

Comment:
We usually compare `err` with `!=nil`. It's odd to do the other way around. Can you modify this piece of code to compare `if err != nil` first? And add a comment to explain why you ignore the error please

Reply:
Silly copy past from another remove autocomplete my bad. I'll fix the other too